features/call/impl/src/main/kotlin/io/element/android/features/call/impl/ui/CallScreenView.kt

```diff
@@ -161,6 +167,10 @@ private fun CallWebView(
                 }
             },
             onRelease = { webView ->
+                // Reset audio mode
+                val audioManager = webView.context.getSystemService<AudioManager>()
+                audioManager?.mode = AudioManager.MODE_NORMAL
```

Comment:
Maybe store the previous mode and restore it here?

Reply:
From what I saw in some issues (this is poorly documented, AFAICT) the app can be 'blacklisted' if it doesn't play any audio in the 6s after enabling this mode or doesn't return to the `MODE_NORMAL` when it's done playing.

Reply:
OK, thanks.